findPhotoDupes

```diff
@@ -254,7 +291,7 @@ if __name__ == '__main__':
             ext2 = os.path.splitext(filename2)[1].lower()
             if ext1 == ext2:
                 # only process if the types are the same
```

Comment:
@jonathankoren this is the fix. `hash` means PIL recognizes the file as image, right? So, we should start the duplicate detection process in that case, not the opposite.

Am I right or misunderstood something? With this change, I successfully detected duplicated images.

Reply:
So even without your fix, it detects duplicates.

In the original code, `getHash()` is going to return a pair that indicates the hash type (sha1 or phash) and the hash value. So in original (left) 249 we check to see if we've seen that pair before. If we did we have a potential duplicate. We do a quick check on the file extensions (so we will allow foo.jpg and foo.jpg.orig to coexist). If the extensions are the same, we check the images dimensions and then flag them as dupes if their the same size.

In `getHash()` we use PIL and imagehash to compute the pHash. If the PIL or image hash can't understand the file (say, it's a video), we fall back to sha1.

Reply:
Here's an example run from laptop

```
SKIPPING photos/2019/2019-02-27/img_9266.mov
CHECKING photos/2019/2019-02-27/img_9266-copy.jpg and photos/2019/2019-02-27/img_9266.jpg
DUPE photos/2019/2019-02-27/img_9266-copy.jpg and photos/2019/2019-02-27/img_9266.jpg
DELETE photos/2019/2019-02-27/img_9266-copy.jpg
```